app/flatpak-builtins-create-usb.c

```diff
@@ -617,7 +622,8 @@ flatpak_builtin_create_usb (int argc, char **argv, GCancellable *cancellable, GE
       }
 
       /* Add dependencies and related refs */
-      if (!add_runtime (all_refs, all_collection_ids, first_ref, dir, cancellable, error))
+      if (!(first_ref_kind & FLATPAK_KINDS_RUNTIME) &&
+          !add_runtime (all_refs, all_collection_ids, first_ref, dir, cancellable, error))
```

Comment:
It looks like the original two lines are repeated immediately below - was that a mistake initially? If so, I think they should be cut (ideally, in a preceding commit).

Reply:
Do you mean where related refs are being added?